docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -231,3 +172,19 @@ A number of advanced topics that were discussed here have been moved out, see be
 === User managed cache
 
 See <<usermanaged.adoc#,the user managed cache documentation>> for more information on this feature.
+
+=== Byte-sized heap
+
+See <<tiering.adoc#byte-sized-heap,the tiering documentation>> for more information on this feature.
+
+=== Update ResourcePools
+
+See <<tiering.adoc#update-resourcepools,the tiering documentation>> for more information on this feature.
```

Comment:
Should this and the two following ones be changed to:
> See the relevant section in the tiering documentation for more information on this feature.

with _relevant section_ being the link.